arangod/Agency/Store.h

```diff
@@ -155,10 +149,6 @@ class Store {
   /// @brief Remove time to live entries for uri
```

Comment:
As far as I see `applies` is only used in `Store.cpp` and could thus be private. Please check.

Reply:
Yes, testing shows that the method is not needed outside of the class. Removing this one too.